subsys/bluetooth/audio/Kconfig.cap

```diff
@@ -48,7 +48,13 @@ config BT_CAP_INITIATOR
 
 config BT_DEBUG_CAP_INITIATOR
 	bool "Common Audio Profile Initiator debug"
+	select DEPRECATED
```

Comment:
Is it needed to select `DEPRECATED` if the parent of Kconfig.cap - `BT_AUDIO` is marked as `EXPERIMENTAL`?

Reply:
That is a good question. It is what had been done in https://github.com/zephyrproject-rtos/zephyr/pull/51897 for all LE Audio implementations, but I agree that the `deprecated` part could/should have been omitted for EXPERIMENTAL features.